<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>miP</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="f_create"/>
    <addaction name="f_open"/>
    <addaction name="f_save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="i_brush"/>
    <addaction name="i_eraser"/>
    <addaction name="separator"/>
    <addaction name="i_line"/>
    <addaction name="i_rect"/>
    <addaction name="i_oval"/>
    <addaction name="separator"/>
    <addaction name="i_select"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="o_about"/>
    <addaction name="o_log"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Цвет 1</string>
     </property>
     <addaction name="c1_black"/>
     <addaction name="c1_red"/>
     <addaction name="c1_blue"/>
     <addaction name="c1_yellow"/>
     <addaction name="c1_green"/>
     <addaction name="c1_white"/>
     <addaction name="separator"/>
     <addaction name="c1_other"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Цвет 2</string>
     </property>
     <addaction name="c2_black"/>
     <addaction name="c2_red"/>
     <addaction name="c2_blue"/>
     <addaction name="c2_yellow"/>
     <addaction name="c2_green"/>
     <addaction name="c2_white"/>
     <addaction name="separator"/>
     <addaction name="c2_other"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
    <addaction name="c_size"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="f_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="f_open">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="f_create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="f_clear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="i_brush">
   <property name="text">
    <string>Кисть</string>
   </property>
  </action>
  <action name="i_eraser">
   <property name="text">
    <string>Ластик</string>
   </property>
  </action>
  <action name="i_line">
   <property name="text">
    <string>Линия</string>
   </property>
  </action>
  <action name="i_rect">
   <property name="text">
    <string>Прямоугольник</string>
   </property>
  </action>
  <action name="i_oval">
   <property name="text">
    <string>Овал</string>
   </property>
  </action>
  <action name="i_triangle">
   <property name="text">
    <string>Треугольник</string>
   </property>
  </action>
  <action name="i_polygon">
   <property name="text">
    <string>Многоугольник</string>
   </property>
  </action>
  <action name="o_property">
   <property name="text">
    <string>Свойства</string>
   </property>
  </action>
  <action name="o_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Выделение</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="i_select">
   <property name="text">
    <string>Выделить</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="o_log">
   <property name="text">
    <string>Лог</string>
   </property>
  </action>
  <action name="c_size">
   <property name="text">
    <string>Размер</string>
   </property>
  </action>
  <action name="c1_red">
   <property name="text">
    <string>Красный</string>
   </property>
  </action>
  <action name="c1_blue">
   <property name="text">
    <string>Синий</string>
   </property>
  </action>
  <action name="c1_yellow">
   <property name="text">
    <string>Желтый</string>
   </property>
  </action>
  <action name="c1_black">
   <property name="text">
    <string>Черный</string>
   </property>
  </action>
  <action name="c1_green">
   <property name="text">
    <string>Зеленый</string>
   </property>
  </action>
  <action name="c1_other">
   <property name="text">
    <string>Другой цвет</string>
   </property>
  </action>
  <action name="c2_black">
   <property name="text">
    <string>Черный</string>
   </property>
  </action>
  <action name="c1_white">
   <property name="text">
    <string>Белый</string>
   </property>
  </action>
  <action name="c2_red">
   <property name="text">
    <string>Красный</string>
   </property>
  </action>
  <action name="c2_blue">
   <property name="text">
    <string>Синий</string>
   </property>
  </action>
  <action name="c2_yellow">
   <property name="text">
    <string>Желтый</string>
   </property>
  </action>
  <action name="c2_green">
   <property name="text">
    <string>Зеленый</string>
   </property>
  </action>
  <action name="c2_white">
   <property name="text">
    <string>Белый</string>
   </property>
  </action>
  <action name="c2_other">
   <property name="text">
    <string>Другой цвет</string>
   </property>
  </action>
  <action name="i_trian">
   <property name="text">
    <string>Треугольник</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
